### DISCUSSION TITLE
Instalando TFEnv - Ambiente Windows

### DISCUSSION THREAD

**Discussion by admpires** [chgasparoto/curso-aws-com-terraform#54] [Q&A]
Boa tarde! Não estou conseguindo instalar o TFEnv no ambiente windows 64bit. Qual o passo a passo para eu poder efetuar esta instalação?

Fico no breve aguardo.

Sds,
Valmir Pires

**Comment by chgasparoto** [comment]
Olá @admpires , eu consegui instalar usando os passos descritos no repositório do [TFEnv](https://github.com/tfutils/tfenv#manual). Eu precisei instalar o [git-bash](https://git-scm.com/) para conseguir rodá-lo corretamente.

> 1. Check out tfenv into any path (here is ${HOME}/.tfenv)
> `$ git clone https://github.com/tfutils/tfenv.git ~/.tfenv`
> 2. Add ~/.tfenv/bin to your $PATH any way you like
> `$ echo 'export PATH="$HOME/.tfenv/bin:$PATH"' >> ~/.bash_profile`

**Reply by andreluiz-barbosa** [reply]
Olá, eu instalei o git-bash, rodei estes comandos, mas o comando tfenv não é reconhecido no Windows...

**Reply by chgasparoto** [reply]
@superbarraca tente instalar este gerenciador que é uma alternativa ao TFEnv
https://tfswitch.warrensbox.com/
